client/login/wp-login/private-site.jsx

```diff
@@ -4,47 +4,38 @@
  * External dependencies
  */
 
-import PropTypes from 'prop-types';
-import React, { Component } from 'react';
-import { localize } from 'i18n-calypso';
+import React from 'react';
+import { useTranslate } from 'i18n-calypso';
 
 /**
  * Internal dependencies
  */
 import Button from 'components/button';
 import Card from 'components/card';
 
-class PrivateSite extends Component {
-	static propTypes = {
-		translate: PropTypes.func.isRequired,
-	};
-
-	render() {
-		const { translate } = this.props;
-
-		return (
-			<Card className="wp-login__private-site">
-				<div className="wp-login__private-site-image">
-					<img src="/calypso/images/private/private.svg" />
-				</div>
-
-				<h2 className="wp-login__private-site-header">
-					{ translate( 'This is a private WordPress.com site.' ) }
-				</h2>
-
-				<p>
-					{ translate(
-						"Request an invitation to view it and we'll " +
-							'send your username to the site owner for their approval.'
-					) }
-				</p>
-
-				<Button primary className="wp-login__private-site-button">
-					{ translate( 'Request Invitation' ) }
-				</Button>
-			</Card>
-		);
-	}
+export default function PrivateSite() {
+	const translate = useTranslate();
+
+	return (
+		<Card className="wp-login__private-site">
+			<div className="wp-login__private-site-image">
+				<img src="/calypso/images/private/private.svg" alt="" />
+			</div>
+
+			<h2 className="wp-login__private-site-header">
+				{ translate( 'This is a private WordPress.com site' ) }
```

Comment:
Hi! I've found a possible matching string that has already been translated **25** times:
`translate( 'This is a private WordPress.com site.' )` *ES Score: 6*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).